Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml

```diff
@@ -378,6 +379,8 @@
   - type: ItemBorgModule
     items:
     - HandheldHealthAnalyzerUnpowered
+    - MedicatedSuture10Lingering
+    - RegenerativeMesh10Lingering
     - Beaker
```

Comment:
Don't just bloat hand count with this stuff, just make it a generic healer instead of this because you'll need to add a dozen to cover every potential damage type.

Reply:
I added these to cover the primary advanced topicals. I understand the bloat concern, which is why I'm not adding things like aloe cream (which is the ONLY other topical medication currently). The thought process / use-case is: "As a medical borg, I should have access to basic/advanced modules which allow me to use standard basic/advanced topical medications respectively".

What others come to mind for you which would create more hand count bloat?

Reply:
Do we actually have a way to create whitelisted Borg hands (that could only hold topicals, for this example)? That may also be useful for the service Borg to be able to hold a shaker that could be inserted / taken from drink dispenser machines if so.

Reply:
unless you mean, by 'generic healer' a new Borg topical which would act more like Omnizine, but that seems a little bit more bloaty than an extra topical in hand since you're creating a whole new item that would also need some discussion probably.

Reply:
> Do we actually have a way to create whitelisted Borg hands (that could only hold topicals, for this example)? That may also be useful for the service Borg to be able to hold a shaker that could be inserted / taken from drink dispenser machines if so.

No :( . From the BorgSystem's 'ProvideItems' method:
![image](https://github.com/space-wizards/space-station-14/assets/12506666/98a82641-463f-4c08-9922-caeec2d5cc1c)